### DISCUSSION TITLE
Allow to Specify Cell Range for Spreadsheets Connector

### DISCUSSION THREAD

**Discussion by rrittsteiger** [airbytehq/airbyte#36307] [Connector Ideas and Features]
Hi, it would be great, if it would be possible to sync named ranges, like Fivetran does. In many situation you don't wan't to sync the whole table. In this cases the easiest way at the moment is to import the needed range in another Spreadsheet and after that import this Spreadsheet via Airbyte. Thank you for all your work.

**Comment by marcosmarxm** [comment]
@rrittsteiger interesting idea today. Let's say what other users have to say about it.

**Comment by AnastasiaGordienko** [comment]
I see a lot of use-cases for this feature, it would be practical and beneficial and would add flexibility to the connector.

[a comment by an automated account is not shown]
